src/clientdlg.cpp

```diff
@@ -298,9 +298,17 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     // Edit menu  --------------------------------------------------------------
     QMenu* pEditMenu = new QMenu ( tr ( "&Edit" ), this );
 
-    pEditMenu->addAction ( tr ( "Clear &All Stored Solo and Mute Settings" ), this, SLOT ( OnClearAllStoredSoloMuteSettings() ) );
+    QMenu* pFaderMenu = new QMenu ( tr ( "Stored &Fader Settings" ), this );
 
-    pEditMenu->addAction ( tr ( "Set All Faders to New Client &Level" ),
+    pEditMenu->addMenu ( pFaderMenu );
```

Comment:
This appears to be the first time Jamulus has used a cascading menu. Works fine on desktop devices, but do we know if it is friendly to Android and iOS?

Reply:
Here's a screenshot from my phone
![Screenshot_20240328-175036](https://github.com/jamulussoftware/jamulus/assets/1549463/ea38b750-93f7-4690-a4ef-8580172ccaad)

Reply:
The iconography isn't clear -- the higher level menu doesn't indicate that the first item is a sub-menu.  But that's par for the course with Qt on Android.